api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationValueTranslatorImplTest.java

```diff
@@ -78,15 +112,12 @@ public void toFhirResource_shouldConvertObsWithCodedValueToCodeableConcept() {
 		concept.setUuid(CONCEPT_VALUE_UUID);
 		obs.setValueCoded(concept);
 		
-		CodeableConcept codeableConcept = new CodeableConcept();
-		codeableConcept.setId(CONCEPT_VALUE_UUID);
-		when(conceptTranslator.toFhirResource(concept)).thenReturn(codeableConcept);
-		
 		Type result = obsValueTranslator.toFhirResource(obs);
 		
 		assertThat(result, notNullValue());
 		assertThat(result, instanceOf(CodeableConcept.class));
-		assertThat(result.getId(), equalTo(CONCEPT_VALUE_UUID));
+		CodeableConcept codeableConceptResult = (CodeableConcept) result;
+		assertThat(codeableConceptResult.getCodingFirstRep().getCode(), equalTo(CONCEPT_VALUE_UUID));
```

Comment:
ACTUAL MEANINGFUL CHANGE @ibacher .  This seems like it was testing for an invalid thing.  I changed the test.  Please review.